src/ICS26Router.sol

```diff
@@ -123,18 +123,30 @@ contract ICS26Router is IICS26Router, IBCStore, Ownable, IICS26RouterErrors, Ree
             value: abi.encodePacked(commitmentBz)
         });
 
-        ics02Client.getClient(msg_.packet.destChannel).membership(membershipMsg);
-        if (msg_.packet.timeoutTimestamp <= block.timestamp) {
-            revert IBCInvalidTimeoutTimestamp(msg_.packet.timeoutTimestamp, block.timestamp);
+        // solhint-disable-next-line no-empty-blocks
+        try ics02Client.getClient(msg_.packet.destChannel).membership(membershipMsg) { }
```

Comment:
Let's remove this for now since it doesn't provide much more utility other than debugging but adds lines of code that require reasoning and linter rules

Reply:
I removed these